db/migrate/20200326220312_drop_favorites_table.rb

```diff
@@ -0,0 +1,9 @@
+class DropFavoritesTable < ActiveRecord::Migration[5.1]
+  def up
+    drop_table :favorites
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
+  end
+end
```

Comment:
Wonder if there's just a way to remove previous migrations.. Even deleting the file and running `rails db:reset` it still treats it like the migration exists and creates the table.